cluster/examples/kubernetes/nfs/common.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name:  rook-nfs-system
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: nfsservers.nfs.rook.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.state
```

Comment:
did you add this manually, i.e. it was not generated from kubebuilder directives?

i have found that when you add additionalPrinterColumns, you lose the default Age column which is a useful one.  If we are specifying these by hand here, I recommend adding this one too:

```
 - JSONPath: .metadata.creationTimestamp
    name: AGE
    type: date
```

If we are generating these, then a kubebuilder directive can be added to types.go like so:
```
// +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
```

This same comment applies to any place we are defining CRDs, we should have consistent printer columns no matter where the CRD is defined/installed from.

Reply:
actually this is generated using [controller-gen](https://github.com/kubernetes-sigs/controller-tools/tree/master/cmd/controller-gen) and also i use kubebuilder markers but i remove it after this is generated.

yeah, i just realized the default Age column was gone after additionalPrinterColumns added (i've reproduced this one). I'll adding Age as additionalPrinterColumns.

I'll also keep kubebuilder markers (eg: validation, subresource, printcolumn) instead removing it, what do you think @jbw976 ?

Reply:
already added AGE column and also i keep kubebuilder markers there instead of removing it.

it can be generated with this command
```
controller-gen paths=./pkg/apis/nfs.rook.io/... crd:trivialVersions=true output:stdout
```